angular/src/providers/menu-controller.ts

```diff
@@ -61,7 +61,17 @@ export class MenuController {
    * @return Returns the instance of the menu, which is useful for chaining.
    */
   swipeEnable(shouldEnable: boolean, menuId?: string): Promise<HTMLIonMenuElement> {
-    return proxyMethod(CTRL, this.doc, 'swipeEnable', shouldEnable, menuId);
+    return proxyMethod(CTRL, this.doc, 'swipeGesture', shouldEnable, menuId);
```

Comment:
I would call swipeGesture() h
```suggestion
    console.warn('MenuController.swipeEnable is deprecated. Use MenuController.swipeGestute() instead');
    return this.swipeGestute(shouldEnable, menuId);
```

Reply:
cc @brandyscarney maybe you have an input on this, how to add deprecation warning